src/test/scala/mockws/MockWSTest.scala

```diff
@@ -131,14 +132,16 @@ class MockWSTest extends FunSuite with Matchers with PropertyChecks {
     val ws = MockWS {
       case (GET, "/url") => Action { Ok("") }
     }
+    import scala.concurrent.ExecutionContext.Implicits.global
+    ws.url("/url2").get() onComplete {
+      case Success(x)=> x.status should be(404)
+      case Failure(x)=> fail("should not throw an exception for url not found")
+    }
```

Comment:
You can use `await` to directly assert the result of the Future:
```
await(ws.url("/url2").get()).status should be (404)
```

Reply:
SOrry, didn't pay attention, it's not Async test.